src/web/nextui/src/app/components/InfoModal.tsx

```diff
@@ -0,0 +1,65 @@
+import BugReportIcon from '@mui/icons-material/BugReport';
+import CalendarTodayIcon from '@mui/icons-material/CalendarToday';
+import ForumIcon from '@mui/icons-material/Forum';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import MenuBookIcon from '@mui/icons-material/MenuBook';
+import { Box, Modal, Typography } from '@mui/material';
+import Link from 'next/link';
+import './InfoModal.css';
+
+export function InfoModal({ open, onClose }: { open: boolean; onClose: () => void }) {
+  return (
+    <Modal open={open} onClose={onClose} className="info-modal">
```

Comment:
Use `Dialog` instead of modal - this will save you a lot of work with the spacing/typography and make it consistent with other dialogs https://mui.com/material-ui/react-dialog/